bootstrap/utils/utils_test.go

```diff
@@ -123,32 +152,82 @@ func TestMergeConfigs(t *testing.T) {
 
 	// verify values
 	assert.True(t, serviceConfig.Writable.StoreAndForward.Enabled)
-	assert.NotEmpty(t, serviceConfig.Writable.StoreAndForward.RetryInterval)
+	assert.Empty(t, serviceConfig.Writable.StoreAndForward.RetryInterval)
 	assert.NotZero(t, serviceConfig.Writable.StoreAndForward.MaxRetryCount)
 	assert.NotEmpty(t, serviceConfig.Trigger.Type)
 }
 
-func TestRemoveZeroValues(t *testing.T) {
-	config := ConfigurationMockStruct{
-		Registry: config.RegistryInfo{
-			Host: "localhost",
-			Port: 8500,
+func TestRemoveUnusedSettings(t *testing.T) {
```

Comment:
Is this supposed to be changed?

```suggestion
func TestRemoveUnusedSettingsFromMap(t *testing.T) {
```

Reply:
TestRemoveUnusedSettings is correct